pkg/restapi/issuer/operation/models.go

```diff
@@ -46,6 +48,7 @@ type WalletConnect struct {
 
 // txnData contains session data.
 type txnData struct {
+	// Todo rename this to profile ID
```

Comment:
yes the issuerID is now misleading it should be profileID added specifically on the value to be renamed. Didnt mention issue number as it can be done part of any follow up prs.